meilisearch-http/public/interface.html

```diff
@@ -203,6 +203,10 @@ <h1 class="title is-1 is-spaced">
 
         if (!Array.isArray(result)) { return }
 
+        result.sort(function(a, b) {
+            return a.name.localeCompare(b.name);
```

Comment:
Beware of performance issues with `String.prototype.localeCompare`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare#performance

You'd may rather use Intl.Collator: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator

Hope to help :)